prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -1198,5 +1198,27 @@ public async Task<ActionResult> FlagSite(int siteId, FromBodyData<bool> flagged)
             await _communitySiteService.UpdateSiteFlag(siteId, flagged);
             return Ok(site);
         }
+
+        // GET: api/sites/5/individual-device-providers
+        /// <summary>
+        /// Gets the Individual Device Providers for a Device Provider Site.
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpGet("{siteId}/individual-device-providers", Name = nameof(GetIndividualDeviceProviders))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<IndividualDeviceProviderViewModel>>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetIndividualDeviceProviders(int siteId)
+        {
+            var site = await _communitySiteService.GetSiteAsync(siteId);
+            if (site == null)
+            {
+                return NotFound($"Site not found with id {siteId}");
+            }
+
+            var providers = await _communitySiteService.GetIndividualDeviceProvidersAsync(siteId);
+            return Ok(providers);
```

Comment:
don't need this in two lines, it can be `return Ok(await _communitySiteService.GetIndividualDeviceProvidersAsync(siteId));`